generator/internal/discovery_to_proto.cc

```diff
@@ -274,6 +274,36 @@ std::vector<DiscoveryFile> CreateFilesFromResources(
   return files;
 }
 
+std::vector<DiscoveryFile> AssignResourcesAndTypesToFiles(
+    std::map<std::string, DiscoveryResource> const& resources,
+    std::map<std::string, DiscoveryTypeVertex> const& types,
+    std::string const& product_name, std::string const& version,
+    std::string const& output_path) {
+  auto files =
+      CreateFilesFromResources(resources, product_name, version, output_path);
+
+  // TODO(#11190): For the first phase of implementation, we create one proto
+  // per resource and its request types. For the remainder of the types, we
+  // dump them all into one internal/common.proto file. This should be reworked
```

Comment:
So is the `internal/` to make sure people don't `#include "google/cloud/foo/v1/internal/common.pb.h"` directly?

(I see that this will be changing, so no big deal anyway. I am just curious.)